src/components/universaldot/DAO/OrganizationTransferOwnershipForm.tsx

```diff
@@ -0,0 +1,103 @@
+import * as Yup from 'yup';
+import merge from 'lodash/merge';
+import { useSnackbar } from 'notistack';
+// form
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+// @mui
+import { Box, Stack, Button, DialogActions } from '@mui/material';
+import { LoadingButton } from '@mui/lab';
+// components
+import { FormProvider, RHFTextField } from '../../hook-form';
+import { useDao } from 'src/hooks/universaldot';
+import { DaoCallables } from 'src/types';
+
+// ----------------------------------------------------------------------
+
+type FormType = {
+  newOwnerId: string;
+};
+
+const getInitialValues = (form: FormType | null) => {
+  const _form = {
+    newOwnerId: '',
+  };
+
+  if (form) {
+    return merge({}, _form, form);
+  }
+
+  return _form;
+};
+
+// ----------------------------------------------------------------------
+
+type FormValuesProps = {
+  newOwnerId: string;
+};
+
+type Props = {
+  form: FormType;
+  onCancel: VoidFunction;
+  organizationId: string;
+};
+
+export default function OrganizationTransferOwnershipForm({
```

Comment:
Got this error when transferring ownershiop also.. 
`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'signAndSend')
    at signedTx (useDao.ts:360:1)`

Reply:
We need to test it together.